Formula/s2geometry.rb

```diff
@@ -0,0 +1,80 @@
+class S2geometry < Formula
+  desc "Computational geometry and spatial indexing on the sphere"
+  homepage "https://github.com/google/s2geometry.git"
+  url "https://github.com/google/s2geometry/archive/v0.9.0-2019.02.11.00.tar.gz"
```

Comment:
This is marked as as “pre-release”, which is not suitable for Homebrew core: https://docs.brew.sh/Acceptable-Formulae

Reply:
We asked for a release tag in https://github.com/google/s2geometry/issues/59. Pre-release is a bit of a stretch, IMHO, as S2 is pretty widely (and wildly) used in the mapping industry. While it may make sense for the authors to tag it as a pre-release, it is rock solid and stable compared to other pieces of software that have much higher release version numbers.

Reply:
I'm using s2geometry in my project and would appreciate this approved very much! Currently I have to download and compile it as a part of my build on OSX.

Reply:
It seems to be a voluntary decision: https://github.com/google/s2geometry/issues/59#issuecomment-465648204

Reply:
Can we then move forward, here?

Reply:
That means it's probably suitable for a tap, but not in core, sorry.

Reply:
> That means it's probably suitable for a tap, but not in core, sorry.

The most recent revision doesn't rely on the explicit pre-release tag of s2, anymore.